doc/sphinx/source/getting_started/config_user.inc

```diff
@@ -86,3 +94,49 @@ plotting such as coastline descriptions and so on.
 Tip: You choose your config.yml file at run time, so you could have several
 available with different purposes. One for formalised run, one for debugging,
 etc...
+
+config-developer.yml
+====================
+
+The config-developer.yml holds the project and machine specific varying part of the path
+to datasets. 
+
+This is necceasiry, because the CMIP data is widely distributed and not all
+machines use the same paths or directory structure. 
+
+If you are using one of the machines listed in the file, then you inform
+ESMValTool about this in the config-user.yml file (described above).
+
+If you are using a machine which is not listed, you can either
+use a flat data directory (ie put all the files in one directory - this is risky and can
+slow down your system). Alternatively, you can add your make your data
```

Comment:
Sound like there is an error in this sentence.

Reply:
```suggestion
slow down your system). Alternatively, you could make your data
```